base/container_scripts/run_integration_tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+declare PROJECT="$1"
+
+set -e
+
+cd "/src/$PROJECT/"
+
+echo "Setting up venv for testing"
+VENVPATH=/tmp/gng_testing
```

Comment:
`$(tempfile -d)` ?
Also a trap to remove it again would be nice.

Reply:
I added --clean-venv for those who want to have the path randomized and cleaned up after every invocation, but I will not set it default to True because that wastes too much time when we're iterating on tests and re-running them over and over.